frontend/src/editor/player.tsx

```diff
@@ -42,38 +43,53 @@ export function PlayerBar({
 
   const [playbackRate, setPlaybackRate] = useLocalStorage('playbackRate', 1);
 
-  const { sources, hasVideo } = useMemo(() => {
+  const { videoSources, audioSources, hasVideo } = useMemo(() => {
     // do not play the original file, it may be large
-    const relevantMediaFiles =
+    let relevantMediaFiles =
       data?.media_files.filter((media) => !media.tags.includes('original')) || [];
 
+    // but if the original is all we have, better play this than nothing at all
+    if (
+      relevantMediaFiles.length == 0 &&
+      data?.media_files !== undefined &&
+      data?.media_files.length > 0
+    ) {
+      relevantMediaFiles = data.media_files;
+    }
```

Comment:
then we could just adjust the sort function to append the original as last item, right?

Reply:
Also we then should prevent the player from loading the (large) original file in background while the loading bee is displayed and the reencode task did not finish yet.

Reply:
done (both)